test/e2e/app/app.go

```diff
@@ -204,6 +217,25 @@ func (app *Application) Info(context.Context, *abci.RequestInfo) (*abci.Response
 	}, nil
 }
 
+// Expected to be called with params set
+func (app *Application) updateBlobMaxBytes(currentHeight int64, params *cmtproto.ConsensusParams) *cmtproto.ConsensusParams {
+	if params == nil {
+		params = &cmtproto.ConsensusParams{}
+	}
```

Comment:
I would move this inside the "if" in line 225, so that not to "disturb" `params` if we're not adding `MaxBytes`